src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example5.java

```diff
@@ -5,15 +5,16 @@
     <property name="minimum" value="1"/>
     <!--  Need to specify a maximum, so 10 times is more than enough. -->
     <property name="maximum" value="10"/>
+    <property name="message"
+                value="File must contain copyright statement"/>
   </module>
 </module>
 */
+package com.puppycrawl.tools.checkstyle.checks.regexp.regexpsingleline;
 
 // xdoc section -- start
-/** // violation, as the file doesn't contain a copyright statement.
-* MyClass as a configuration example.
-*/
-class MyClass {
-
-}
+/** // violation 'File must contain copyright statement'
+ * MyClass as a configuration example.
+ */
+public class Example5 {}
```

Comment:
@romani I'm struggling to get this test to pass. The `verifyWithInlineConfigParser` isn't identifying any violation. I'm tried different combinations (see below) but nothing seems to work.
- `// violation 'blah blah'`
- `// violation below 'blah'`
- `// violation on first line 'blah'`

This is a similar test to Example7 (which passes), with the only difference being file types e.g. sql and java

Reply:
because you have it at line 4 `    <property name="format" value="This file is copyrighted"/>`

make line 4 as `    <property name="format" value="This\ file is copyrighted"/>`, escaping space that is not required but possible by regexp.

Reply:
Let's change it to `<property name="format" value="^[ ]*\*This file is copyrighted"/>`

To validate that it is part of javadoc, leading `*` on a line.

Reply:
Sounds like we are trying to avoid fixing our buggy code

Reply:
> Let's change it to `<property name="format" value="^[ ]*\*This file is copyrighted"/>`

Done

> To validate that it is part of javadoc, leading `*` on a line.

I didn't fully understand this, but implemented what I think you intended. Unfortunately the test still is still failing. Will debug when I have time.